src/commands/Games/coinflip.ts

```diff
@@ -0,0 +1,56 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { MessageEmbed } from 'discord.js';
+import { getColor, cleanMentions } from '../../lib/util/util';
+
+const enum CoinTypes { Heads, Tails }
+export default class extends SkyraCommand {
+
+	private readonly cdnTypes = ['heads', 'tails'] as const;
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['cf'],
+			bucket: 2,
+			cooldown: 7,
+			description: language => language.tget('COMMAND_COINFLIP_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_COINFLIP_EXTENDED'),
+			requiredPermissions: ['ATTACH_FILES'],
+			runIn: ['text'],
+			usage: '<50|100|200|500|1000|2000|5000|10000> <coin:cointype>',
+			usageDelim: ' '
+		});
+
+		this.createCustomResolver('cointype', (arg, possible, message) => {
+			const lArg = arg.toLowerCase();
+			const face = message.language.tget('COMMAND_COINFLIP_COINNAMES').findIndex(coin => coin.toLowerCase() === lArg);
+			if (face === -1) throw message.language.tget('COMMAND_COINFLIP_INVALID_COINNAME', cleanMentions(message.guild!, arg));
+			return face;
+		});
+	}
+
+	public async run(message: KlasaMessage, [bet, guess]: [string, CoinTypes]) {
+		await message.author.settings.sync();
+		const wager = Number(bet);
+		const money = message.author.settings.get(UserSettings.Money);
+
+		if (money < wager) {
+			throw message.language.tget('COMMAND_SLOTMACHINES_MONEY', money);
+		}
+
+		const coinNames = message.language.tget('COMMAND_COINFLIP_COINNAMES');
+
+		const result = Math.random() > 0.5 ? CoinTypes.Heads : CoinTypes.Tails;
+		const won = result === guess;
```

Comment:
This will never work if you translate coins. You'll want to adapt your custom resolver

Reply:
Enums are numbers, and COINNAMES is an array. 0 is always heads, 1 is always tails. I don't see why it wouldn't work

Reply:
Right, you're working with numbers not strings, my bad 😅